fbchat/client.py

```diff
@@ -1779,9 +1773,8 @@ def deletePlan(self, plan):
                 }]
             }
         }
-        url_part = urllib.parse.urlencode(full_data)
 
-        j = self._post('{}/?{}'.format(self.req_url.PLAN_DELETE, url_part), fix_request=True, as_json=True)
+        j = self._post(self.req_url.PLAN_CHANGE, full_data, fix_request=True, as_json=True)
```

Comment:
Hmm, seems like you changed `self.req_url.PLAN_DELETE` to `self.req_url.PLAN_CHANGE` by mistake ;)

Reply:
No, request for deleting and changing was the same, but I forgot about it and when changing Event to Plan I wrote PLAN_DELETE, hah.